src/app/intro/views/MaasIntro/MaasIntro.test.tsx

```diff
@@ -113,24 +120,46 @@ describe("MaasIntro", () => {
     ).toBe(0);
   });
 
-  it("can dispatch actions to update the default package repository urls", () => {
+  it("can dispatch actions to update the default package repository urls", async () => {
     const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter initialEntries={[{ pathname: "/intro", key: "testKey" }]}>
-          <CompatRouter>
-            <MaasIntro />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
+    renderWithMockStore(
+      <MemoryRouter initialEntries={[{ pathname: "/intro", key: "testKey" }]}>
+        <CompatRouter>
+          <MaasIntro />
+        </CompatRouter>
+      </MemoryRouter>,
+      { store }
     );
-    submitFormikForm(wrapper, {
-      httpProxy: "http://localhost:3000",
-      mainArchiveUrl: "http://www.newmainarchive.com",
-      name: "my new maas",
-      portsArchiveUrl: "http://www.newportsarchive.com",
-      upstreamDns: "0.0.0.0",
+    const name = screen.getByRole("textbox", { name: NameCardLabels.Name });
+    const proxy = screen.getByRole("textbox", {
+      name: ConnectivityCardLabels.HttpProxy,
+    });
+    const upstream_dns = screen.getByRole("textbox", {
+      name: ConnectivityCardLabels.UpstreamDns,
+    });
+    const main_archive = screen.getByRole("textbox", {
+      name: ConnectivityCardLabels.MainArchiveUrl,
     });
+    const ports_archive = screen.getByRole("textbox", {
```

Comment:
We normally use camelCase for variable names on the front end.
```suggestion
    const portsArchive = screen.getByRole("textbox", {
```